components/Spinner/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import './spinner.scss'
+
+type SpinnerProps = {
+    id?: string;
+    isActive: boolean;
+    className: string;
+}
+function Spinner(props: SpinnerProps) {
+    return (
+        <>
+            {props?.isActive && (
+                <div
+                    id={(props.id ? props.id : "spinner")}
+                    tabIndex={-1}
+                    aria-hidden="true"
```

Comment:
this is just a question, but how does a screen reader let the user know that the site is loading/processing?

Reply:
Good point. I should probably add some hidden text that is readable by the screen reader that says "loading".